cmd/flow-aggregator/options.go

```diff
@@ -148,21 +137,28 @@ func (o *Options) validate(args []string) error {
 		if err != nil {
 			return err
 		}
-		if o.clickHouseCommitInterval < minClickHouseCommitInterval {
+		if o.clickHouseCommitInterval < flowaggregatorconfig.MinClickHouseCommitInterval {
 			return fmt.Errorf("commitInterval %s is too small: shortest supported interval is %s",
-				o.config.ClickHouse.CommitInterval, minClickHouseCommitInterval)
+				o.config.ClickHouse.CommitInterval, flowaggregatorconfig.MinClickHouseCommitInterval)
 		}
 	}
 	return nil
 }
 
 func (o *Options) loadConfigFromFile(file string) (*flowaggregatorconfig.FlowAggregatorConfig, error) {
```

Comment:
do you think we could just move everything related to the configuration file to `pkg/flowaggregator` (even `validate`)?
Then we can just initialize the flow aggregator as follows:
```go
aggregator.NewFlowAggregator(pathToConfig, ....)
```
I feel like it's better to manage the config (reading / watching) all in one place, given the approach we are going for.

Reply:
I don't think that too much code needs to change. The `NewFlowAggregator` implementation will change as it needs to start the watch, and read the initial config, but that shouldn't be too big of a change?

Reply:
I've moved the whole option.go to pkg/flowaggregator/util. Thanks for suggestions.